guides/configuring_remote_video_subscriptions.md

```diff
@@ -0,0 +1,94 @@
+
+First we can have an observer on [AudioVideoObserver.remoteVideoSourcesDidBecomeAvailable]
+(https://aws.github.io/amazon-chime-sdk-ios/Protocols/AudioVideoObserver.html#/c:@M@AmazonChimeSDK@objc(pl)AudioVideoObserver(im)remoteVideoSourcesDidBecomeAvailableWithSources:) / [[AudioVideoObserver.remoteVideoSourcesDidBecomeUnavailable](https://aws.github.io/amazon-chime-sdk-ios/Protocols/AudioVideoObserver.html#/c:@M@AmazonChimeSDK@objc(pl)AudioVideoObserver(im)remoteVideoSourcesDidBecomeUnavailableWithSources:) and keep track of if which clients are publishing video and store some initial configuration, before calling [AudioVideoFacade.updateVideoSourceSubscriptions](https://aws.github.io/amazon-chime-sdk-ios/Protocols/
+AudioVideoControllerFacade.html#/c:@M@AmazonChimeSDK@objc(pl)AudioVideoControllerFacade(im)updateVideoSourceSubscriptionsWithAddedOrUpdated:removed:) with default configurations.
+
+
+```
+class VideoModel: NSObject {
+    public var remoteVideoSourceConfigurations: Dictionary<RemoteVideoSource, VideoSubscriptionConfiguration> = Dictionary()
+}
+
+func remoteVideoSourcesDidBecomeAvailable(sources: [RemoteVideoSource]) {
+    sources.forEach { source in
+        videoModel.remoteVideoSourceConfigurations[source] = VideoSubscriptionConfiguration()
+    }
+    audioVideoFacade.updateVideoSourceSubscriptions(addedOrUpdated: remoteVideoSourceConfigurations, removed: [])
+}
+
+func remoteVideoSourcesDidBecomeUnavailable(sources: [RemoteVideoSource]) {
+    sources.forEach { source in
+        videoModel.remoteVideoSourceConfigurations.removeValue(forKey: source)
+    }
+    // No need to call `updateVideoSourceSubscription`
+}
+
+```
+
+To update this code to handle the pinned user case, we can use other application code to determine if a video source is pinned, and if so increase the priority above the default. This may be easier if storing RemoteVideoSource in the UI view, or using some other mapping. The RemoteVideoSource used must be the same as the one provided in [AudioVideoObserver.remoteVideoSourcesDidBecomeAvailable]
+(https://aws.github.io/amazon-chime-sdk-ios/Protocols/AudioVideoObserver.html#/c:@M@AmazonChimeSDK@objc(pl)AudioVideoObserver(im)remoteVideoSourcesDidBecomeAvailableWithSources:). *Do not make new `RemoteVideoSources` from scratch.* 
+
+```
+fun pinTile(attendeeId: String) {
+    for (source, config) in remoteVideoSourceConfigurations {
+        if attendeeId == source.attendeeId {
+            config.priority = VideoPriority.highest
+        }
+    }
+ 
+    audioVideo.updateVideoSourceSubscriptions(remoteVideoSourceConfigurations, emptyArray())
```

Comment:
technically this could go on line 85 as:
```
     audioVideo.updateVideoSourceSubscriptions([config], []) // i probably have array syntax wrong
     ````

Reply:
`config` is the value in `remoteVideoSourceConfigurations`, we want to pass the map in `updateVideoSourceSubscriptions`